### PR TITLE
Refactor queueManager to introduce in-progress meeting concept (#228, #230)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR refactors `queueManager.tsx` (moving code related to meeting editing and meeting tables into a new module, `meetingTables.tsx`) in order to allow for a separate in-progress meetings section on the Manage page. Some of the code (particularly related to the meeting type dial-in info) is preliminary. Work on related attendee-facing changes will be tackled next. The PR aims to resolve issues #228 and #230.

### CODE REVIEW

File: src/assets/src/models.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export interface Meeting {
     backend_type: "bluejeans"|"zoom"|"inperson";
     backend_metadata?: BluejeansMetadata|ZoomMetadata;
     created_at: string;
+    status: 0 | 1 | 2;  // 0 is UNASSIGNED, 1 is ASSIGNED, 2 is STARTED
```
Comment (jlost) [inline src/assets/src/models.ts:36]: I think an enum could be used here.

**Review by jlost — APPROVED** [review]
Looks good!

File: src/assets/src/models.ts (hunk 1 above)
Comment (ssciolla) [reply src/assets/src/models.ts:36]: So like this?

```
enum MeetingStatus {
   0,
   1,
   2
}
```

Comment (jlost) [reply src/assets/src/models.ts:36]: I was thinking:
```
enum MeetingStatus {
        UNASSIGNED = 0,
        ASSIGNED = 1,
        STARTED = 2,
}
```
Which results in (from node repl):
```
> MeetingStatus
{
  '0': 'UNASSIGNED',
  '1': 'ASSIGNED',
  '2': 'STARTED',
  UNASSIGNED: 0,
  ASSIGNED: 1,
  STARTED: 2
}
```

Comment (ssciolla) [reply src/assets/src/models.ts]: Mmm, yes, so like this? https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/240/commits/8fec90a9f5db8a79cefca5d30a7a48120aa9e3f2

**Comment by ssciolla** [comment]
Thanks, @jlost, I'm merging this so we can keep moving.

**Pull request merged**